install failing on python 3.7.0

## Steps to reproduce:
- Create a fresh python 3.7.0 virtualenv
- Activate the virtualenv
- Run `pip install yaml-config`
## Expected behavior:
- Install succeeds and module is available within virtualenv
## Experienced behavior:
- The below error is shown when PyYAML is installed:
```
Running setup.py install for PyYAML ... error
    Complete output from command ...python -u -c "import setuptools, tokenize;__file__='/private/var/folders/zq/983h18ss4qj9ydkf873wx_nc0000gn/T/pip-install-8bhgz4cg/PyYAML/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/var/folders/zq/983h18ss4qj9ydkf873wx_nc0000gn/T/pip-record-5d90keuk/install-record.txt --single-version-externally-managed --compile --install-headers ...python3.7/PyYAML:
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.13-x86_64-3.7
    creating build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/scanner.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/error.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/constructor.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/composer.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/events.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/__init__.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/representer.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/tokens.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/dumper.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/cyaml.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/parser.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/reader.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/loader.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/resolver.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/serializer.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/nodes.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    copying lib3/yaml/emitter.py -> build/lib.macosx-10.13-x86_64-3.7/yaml
    running build_ext
    creating build/temp.macosx-10.13-x86_64-3.7
    checking if libyaml is compilable
    [...]
    In file included from ext/_yaml.c:271:
    ext/_yaml.h:10:9: warning: 'PyString_CheckExact' macro redefined [-Wmacro-redefined]
    #define PyString_CheckExact PyBytes_CheckExact
            ^
    ext/_yaml.c:139:11: note: previous definition is here
      #define PyString_CheckExact          PyUnicode_CheckExact
              ^
    ext/_yaml.c:1410:17: warning: assigning to 'char *' from 'const char *' discards qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
    [...]
...pystate.h:238:15: note: 'curexc_traceback' declared here
        PyObject *curexc_traceback;
                  ^
    51 warnings and 15 errors generated.
    error: command 'clang' failed with exit status 1

    ----------------------------------------
```
Currently the pinned version of `PyYAML` is 3.12 https://github.com/GreenBuildingRegistry/yaml-config/blob/dev/setup.cfg#L24
This issue gets fixed in version 3.13 https://github.com/yaml/pyyaml/issues/152
Adjusting the version number fixes the above install error and all the tests in the repo pass 100%.

I opened a PR here to add the fix https://github.com/GreenBuildingRegistry/yaml-config/pull/3

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+0.1.1 [2018-09-04]
+------------------
+* Add python 3.7.0 compatibility
+
 0.1.0 [2018-02-16]
 ------------------
 * OpenSource release

--- a/requirements/dev.txt
+++ b/requirements/dev.txt
@@ -1,5 +1,5 @@
 -r base.txt
-coverage==4.2
+coverage>=4.2
 flake8>=3.0.4
 frosted>=1.4.1
 isort>=4.2.5

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,6 +1,6 @@
 [metadata]
 name=yaml-config
-version=0.1.0
+version=0.1.1
 description=Python client for reading yaml based config files
 author=Paul Munday
 author_email=paul@paulmunday.net
@@ -16,12 +16,13 @@ classifiers =
 	Programming Language :: Python :: 3.4
 	Programming Language :: Python :: 3.5
 	Programming Language :: Python :: 3.6
+	Programming Language :: Python :: 3.7
 [options]
 packages = find:
 include_package_data = True
 zip_safe = False
 install_requires =
-	PyYAML==3.12
+	PyYAML==3.13
 	typing==3.6.1
 [bdist_wheel]
 universal = 1

--- a/tox.ini
+++ b/tox.ini
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py34,py35,py36
+envlist = py27, py34,py35,py36,py37
 
 [testenv]
 deps=

